feature_apps/src/main/java/cm/aptoide/pt/feature_apps/presentation/ViewModelProvider.kt

```diff
@@ -58,10 +54,7 @@ fun rememberApp(
 )
 
 @Composable
-fun rememberAppBySource(
-  source: String,
-  useStoreName: Boolean = true,
-): Pair<AppUiState, () -> Unit> = runPreviewable(
+fun rememberAppBySource(source: String): Pair<AppUiState, () -> Unit> = runPreviewable(
```

Comment:
Lets make the same migration for `v10` and remove this function, leaving only `rememberApp` that returns by source